public/Aframe/samples/number_lock.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>CoronaEscape</title>
+        <meta name="description" content="CoronaEscape" />
+        <script src="https://aframe.io/releases/1.1.0/aframe.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/gh/donmccurdy/aframe-extras@v6.1.0/dist/aframe-extras.min.js"></script>
+        <!-- <script src="https://rawgit.com/aframevr/aframe/c4aa63e/dist/aframe-master.min.js"></script> -->
+        <script src="https://unpkg.com/aframe-environment-component@1.0.0/dist/aframe-environment-component.min.js"></script>
+        <script src="aframe-super-keyboard.min.js"></script>
+        <script src="keypad.js"></script>
+        <script src="floor.js"></script>
+        <script src="//cdn.rawgit.com/donmccurdy/aframe-physics-system/v4.0.1/dist/aframe-physics-system.min.js"></script>
+    </head>
+    <body>
+        <a-scene
+            cursor="rayOrigin:mouse"
+            background="color: #ECECEC"
+            physics="debug: false; gravity: -9.81"
+            raycaster="objects: .link"
+        >
+            <a-box
```

Comment:
I think all the floor stuff and gravity/physics stuff can be removed from this PR, as we want to compartmentalize this PR to only numpad component, that can be added to our scenes once we start working on those